src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -262,47 +260,45 @@ export class TerminalService implements ITerminalService {
 	}
 
 	handleNewRegisteredBackend(backend: ITerminalBackend) {
```

Comment:
was there a reason we were only doing this for remote terminals before @Tyriar ?

Reply:
don't think so as the local terminal promise was within the block